src/amo/components/AddonsByAuthorsCard/index.js

```diff
@@ -359,25 +373,29 @@ export class AddonsByAuthorsCardBase extends React.Component<InternalProps> {
 export const mapStateToProps = (state: AppState, ownProps: Props) => {
   const { addonType, authorIds, forAddonSlug, numberOfAddons } = ownProps;
 
-  let addons = getAddonsForAuthorIds(
-    state.addonsByAuthors,
-    authorIds,
-    addonType,
-    forAddonSlug,
-  );
-  addons = addons ? addons.slice(0, numberOfAddons) : addons;
+  let addons = [];
```

Comment:
I don't have a change request but I want to point out that initializing this to an empty array is risky. An empty array should be reserved for the case when the API returned an empty set. Before an API fetch has been made, the value should be `undefined` or `null`.

These issues will come to light when you try to make it [load](https://github.com/mozilla/addons-frontend/blob/0b6d7c48c5a13733833e5795925101ecfb57397e/src/amo/components/AddonsByAuthorsCard/index.js#L97) only once on the server in a future patch so I guess you could wait until then. Right now the selector and probably other things would need changes to support conditional loading.

Reply:
Yep, good call. I'll change that